samples/rtl/add_fortytwo_vec.py

```diff
@@ -0,0 +1,194 @@
+import dace
```

Comment:
Please add a description of what this sample shows. Maybe we don't need both this and the non-vec one?

Reply:
Fixed in https://github.com/carljohnsen/dace/commit/11984d3c2c997860d2124ef82133077ccd84e753